### ISSUE TITLE
a11y/media-prefers-reduced-motion: doesn't work with mixins

### ISSUE THREAD

**Issue by mayank99** [YozhikM/stylelint-a11y#63]
We have a lot of code in our project that does something like this:

```scss
@mixin some-mixin {
  transition: all 0.2s ease-out;
  // ... other code
}
```

and we use those mixins like so:
```scss
.some-class {
  @include some-mixin;
  // ... other code
}
```

It would be useful if the plugin could error out either inside the mixin or where it's used.
